plugin.py

```diff
@@ -723,6 +723,10 @@ def _do_karma(self, irc, channel, agent, recip, line, explicit=False):
         # Extract 'puiterwijk' out of 'have a cookie puiterwijk++'
         recip = recip.strip().split()[-1]
 
+        # Exclude 'c++', 'g++' or 'i++' (c,g,i), issue #30
+        if len(recip) == 1 and recip.lower() in ['c','g','i']:
```

Comment:
Thanks @pypingou I understand all suggestions, I tested it with my script, and learned something ;-)
Also I updated the branch with your suggestions!